sdks/java/core/src/main/java/org/apache/beam/sdk/coders/RowCoderGenerator.java

```diff
@@ -99,37 +97,14 @@
 @Experimental(Kind.SCHEMAS)
 public abstract class RowCoderGenerator {
   private static final ByteBuddy BYTE_BUDDY = new ByteBuddy();
-  private static final ForLoadedType CODER_TYPE = new ForLoadedType(Coder.class);
-  private static final ForLoadedType LIST_CODER_TYPE = new ForLoadedType(ListCoder.class);
-  private static final ForLoadedType ITERABLE_CODER_TYPE = new ForLoadedType(IterableCoder.class);
-  private static final ForLoadedType MAP_CODER_TYPE = new ForLoadedType(MapCoder.class);
   private static final BitSetCoder NULL_LIST_CODER = BitSetCoder.of();
   private static final VarIntCoder VAR_INT_CODER = VarIntCoder.of();
-  private static final ForLoadedType NULLABLE_CODER = new ForLoadedType(NullableCoder.class);
 
   private static final String CODERS_FIELD_NAME = "FIELD_CODERS";
 
-  // A map of primitive types -> StackManipulations to create their coders.
-  private static final Map<TypeName, StackManipulation> CODER_MAP;
-
   // Cache for Coder class that are already generated.
   private static Map<UUID, Coder<Row>> generatedCoders = Maps.newConcurrentMap();
```

Comment:
nit: not directly relevant to this PR, but I guess this field should be final, and name should follow convention

Reply:
done